src/main/java/pascal/taie/analysis/pta/core/solver/DefaultSolver.java

```diff
@@ -354,12 +354,11 @@ private void processInstanceLoad(CSVar baseVar, PointsToSet pts) {
         Var var = baseVar.getVar();
         for (LoadField load : var.getLoadFields()) {
             Var toVar = load.getLValue();
-            JField field = load.getFieldRef().resolve();
             if (isConcerned(toVar)) {
                 CSVar to = csManager.getCSVar(context, toVar);
                 pts.forEach(baseObj -> {
                     InstanceField instField = csManager.getInstanceField(
-                            baseObj, field);
+                            baseObj, load.getFieldRef().resolve());
```

Comment:
```suggestion
                JField field = load.getFieldRef().resolve();
                pts.forEach(baseObj -> {
                    InstanceField instField = csManager.getInstanceField(baseObj, field);
```

Reply:
I'm suggesting this changes. (Similarly, changes need to be made in both `processInstanceLoad` and `processInstanceStore`)